mobile/mobile.go

```diff
@@ -653,3 +622,11 @@ func parseKeys(nameSpace, localPrivKey,
 
 	return localStaticKey, remoteStaticKey, nil
 }
+
+func init() {
+	var err error
+	permsMgr, err = perms.NewManager(true)
```

Comment:
Same here, not sure if the mobile framework calls `init()`. Could be in `InitLNC` (after the mutex, with a nil check in case it was already initialized).